app/design/frontend/Ludmila/ludmila_theme/Magento_Theme/layout/default.xml

```diff
@@ -0,0 +1,15 @@
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="logo">
+            <arguments>
+                <argument name="logo_img_width" xsi:type="number">70</argument>
```

Comment:
Cannot see the custom logo file specified. Only logo size was changed. Please add your logo

Reply:
Cannot see the custom logo file specified. Only logo size was changed. Please add your logo